src/explore-education-statistics-frontend/src/modules/data-catalogue/components/__tests__/Filters.test.tsx

```diff
@@ -97,12 +107,13 @@ describe('Filters', () => {
   });
 
   test('calls the onChange handler when the theme filter is changed', async () => {
+    const user = userEvent.setup();
```

Comment:
This is quite repetitive. Couldn't we stick this in our common `render` function?

For example:

```ts
// render.tsx

interface CustomRenderResult extends RenderResult {
  user: UserEvent;
}

/**
 * Custom test render function that is pre-configured with any contexts
 * that would otherwise create unnecessary boilerplate.
 */
export default function render(
  ui: ReactElement,
  options?: Omit<RenderOptions, 'queries'>,
): CustomRenderResult {
  return {
    ...(baseRender(ui, {
      wrapper: DefaultWrapper,
      ...options,
    }) as RenderResult),
    user: userEvent.setup(),
  };
}
```

We'd then be able to use it like:

```
const { user } = render(<Blah/>); 
```

This is something they suggest in the docs as well: https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent